### ISSUE TITLE
Qt client, accessibility bug: when you open any dialogue in the app, the NVDA screen reader announces that dialogue twice

### ISSUE THREAD

**Issue by amirmahdifard** [BearWare/TeamTalk5#1720]
### Description

in the Qt client, if you open any dialogue, server properti, settings dialogue, user volume dialogue, user information dialogue, text message dialogue, user account dialogue, banned user dialogue, connect to a server dialogue and so on, when you open them, NVDA announces the title of that dialogue, the items of that dialogue and everything twice, this can a little bit make the screen reader talk more when it doesn't need too, and slo it down a little bit: i will example one of them with opening text message dialogue as an example in the next line, but be noted that this bug happens with all dialogues in the app
Messages - Music Player  dialog  Private chat with Music Player Message  grouping Message  edit  blank Messages - Music Player  dialog  Private chat with Music Player Message  grouping Message  edit  blank
version: 5.12 stable and 5.13 beta

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkClassic
- [ ] TeamTalkServer

### Platform

- [X] Windows
- [ ] macOS
- [ ] Android
- [ ] iOS
- [ ] Linux

### Expected behavior

the screen reader should announce the title and the items of that dialogue only once, like this.
Messages - Music Player  dialog  Private chat with Music Player Message  grouping Message  edit  blank

### Actual behavior

the screen reader announces the title and the items twice, resulting on screen reader freez sometimes, and slow down a little bit when opening a dialogue, like this.
Messages - Music Player  dialog  Private chat with Music Player Message  grouping Message  edit  blank Messages - Music Player  dialog  Private chat with Music Player Message  grouping Message  edit  blank

### Steps to reproduce problem

1: open Qt teamtalk
2: open some dialogues, such as connect to a server dialogue and so on.

**Comment by CoBC** [comment]
I suspect this one to be related to NVDA, so I've reported here:
[NVAccess/NVDA Issue 14560](https://github.com/nvaccess/nvda/issues/14560)

**Comment by amirmahdifard** [comment]
probably not gonna be fixed anyway. like other Qt accessibility bugs, saying down at the end of combo box, saying collapsed in files list and so on.
